index.html

```diff
@@ -0,0 +1,242 @@
+                               <div class="comment-feedback">
+                                    <i class="material-icons">thumb_up</i>
+                                    <i class="material-icons">thumb_down</i>
+                                    <a href="" class="btn-link">REPLY</a>
+                               </div>
+                            </div>
+                        </div>
+                        <div class="comment flex">
+                            <div class="user-thumbnail"><img src="./images/img_avatar.png" class="avatar" alt="User Profile"></div>
+                            <div>
+                                <span class="comment-header"><span class="header-5">Bello</span> <span class="fade">1 day ago</span></span>
+                               <p>Anyone else wish alan walker would respond and give a heart to your comment</p>
+                               <div class="comment-feedback">
+                                    <i class="material-icons">thumb_up</i>
+                                    <i class="material-icons">thumb_down</i>
+                                    <a href="" class="btn-link">REPLY</a>
+                               </div>
+                            </div>
+                        </div>
+                        <div class="comment flex">
+                            <div class="user-thumbnail"><img src="./images/img_avatar.png" class="avatar" alt="User Profile"></div>
+                            <div>
+                                <span class="comment-header"><span class="header-5">Tshepo</span> <span class="fade">1 day ago</span></span>
+                               <p>Some text and more text for you</p>
+                               <div class="comment-feedback">
+                                    <i class="material-icons">thumb_up</i>
+                                    <i class="material-icons">thumb_down</i>
+                                    <a href="" class="btn-link">REPLY</a>
+                               </div>
+                            </div>
+                        </div>
+                        <div class="comment flex">
+                            <div class="user-thumbnail"><img src="./images/img_avatar.png" class="avatar" alt="User Profile"></div>
+                            <div>
+                                <span class="comment-header"><span class="header-5">Bello</span> <span class="fade">1 day ago</span></span>
+                               <p>Some text and more text for you</p>
+                               <div class="comment-feedback">
+                                    <i class="material-icons">thumb_up</i>
+                                    <i class="material-icons">thumb_down</i>
+                                    <a href="" class="btn-link">REPLY</a>
+                               </div>
+                            </div>
+                        </div>
+
+                    </div>
+                </div>
```

Comment:
The images in this section aren't working. 

![Screen Shot 2020-07-13 at 8 01 01 PM](https://user-images.githubusercontent.com/48804258/87337346-94e90000-c543-11ea-9375-d7e9565a1287.png)